packages/@okta/vuepress-site/docs/guides/configure-signon-policy/next-steps/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Next steps
+---
+
+You should now understand how to add an <GuideLink link="../prompt-factor-group">Okta Sign-On Policy</GuideLink> and an <GuideLink link="../prompt-factor-outside-us">App Sign-On Policy</GuideLink>.
+
+Next, take a look at how to sign users out of Okta and out of an app, configuring self-registration, and configuring an access policy:
```

Comment:
Do we prefer "app" or "application"? Further, do we try to be consistent? I don't have an opinion in this matter and just want to know for my own sake.

Reply:
We try to use App